SpeechResponder/Help.md

```diff
@@ -203,6 +203,23 @@ Common usage of this is to provide clear callsigns and idents for ships, for exa
 
    Ship ident is {ICAO(ship.ident)}.
 
+### JumpDetails()
+
+This function will provide jump information based on your ship loadout and current fuel level, dependent on the followiing types:
+
+  * `next` range of next jump at current fuel level
+  * `max` range of single jump at 'max fuel'
+  * `total` total range of multiple jumps at current fuel level
```

Comment:
`max` may be confusing... I think people will expect it to match the maximum jump range of their ship (zero cargo, only only fuel for one max range jump). Instead, it calculates jump range based on max fuel consumed but including the mass of the fuel left in the tanks and cargo so it doesn't match the max jump range shown in the right hand panel of the ship.

I'm not sure `total` is needed? `full` provides a better number.

Reply:
Changed `max` description to `range of single jump at 'max fuel' and current laden mass`.

Again, I gave this considerable thought, to the extent that I used `total` in the example below. `Full` will not tell you if your current fuel level is sufficient to make it to a particular destination.